content/md/products-super-power/work-on-your-assets.md

```diff
@@ -72,6 +72,23 @@ You also have the completeness indicator directly on the asset thumbnail in the
 
 The completeness is **automatically calculated** as soon as we enrich a required attribute. It means that the bullet point will disappear as soon as the required attribute is filled, and same for the completeness percentage evolution.
 
+# Edit assets
+## Bulk edit a selection of asset attributes
+It is possible to bulk edit your own selection of asset attributes in an asset family.
+1. Check the box under each asset you would like to bulk edit
+1. In the bottom bar, click on `edit`
+1. At the "edit" step, click on `add attributes` in the top right menu
+![Assets attributes selection](../img/Assets_bulk-edit-add-attributes.png)
+1. Select the attributes you would like to bulk edit
+![Assets attributes selection](../img/Assets_bulk-edit-attributes-selection.png)
+1. Fill in the values you want for each selected attribute
+1. Once you are done, click on `next` in the top right corner
+1. At the "confirm" step, you can visualise the changes that will be made. If you want to continue, click on `Confirm`
+
+::: warning
+Please note that it is possible to add the same attribute multiple times. For instance, if you would like to edit an attribute that is scopable and/or localisable, you will have the possibility to do so by adding the attribute as many times as you want, but you will have to carefully select a different locale and/or scope for each of them so that you will avoid to setting different values for the same attribute with the same locale and/or same scope.
```

Comment:
```suggestion
Please note that it is possible to add the same attribute multiple times. For instance, if you would like to edit an attribute that has a different value per channel and/or per locale, you will have the possibility to do so by adding the attribute as many times as you want, but you will have to carefully select a different locale and/or channel for each of them so that you will avoid setting different values for the same attribute with the same locale and/or same channel.
```

Reply:
Je pense qu'il faudrait simplifier la phrase mais ça sort de mes compétences :D (à voir avec Camille)